public/src/js/models/article/editor.js

```diff
@@ -56,16 +56,24 @@ export default class Editor extends BaseClass {
             return opts.maxLength && this.value().length > opts.maxLength;
         });
         this.displayEditor = ko.pureComputed(this.isVisible, this);
-        this.updateText = ko.pureComputed({
-            read: this.value,
-            write: this.writeText,
-            owner: this
-        });
+        if (type === 'text') {
+            this.updateText = ko.pureComputed({
+                read: this.value,
+                write: this.writeText,
+                owner: this
+            });
+        } else if (type === 'text-image') {
+            this.updateText = ko.pureComputed({
+                read: this.imageValue,
+                write: this.writeImage,
+                owner: this
+            });
+        }
 
-        if (validator && _.isFunction(this[validator.fn])) {
```

Comment:
We need to keep both, `validator && _.isFunction(this[validator.fn])` is still used by cutout images